api/setup/setup.go

```diff
@@ -69,6 +70,12 @@ func Schemas(ctx context.Context, management *config.ManagementContext, schemas
 
 	NamespacedTypes(schemas)
 
+	secretStore, err := machineconfig.NewStore(management)
```

Comment:
You want to setup all the types before we setup the crdStore.  Can you move this back up before L52?